src/Autocomplete/src/Checksum/ChecksumCalculator.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
Not usually set in symfony components AFAIK

(sorry for the c/c)

Reply:
@weaverryan should I remove it? I've added it as I have it in my PHP class template in the IDE (as I consider it as a good practice) 🤔.